syft/execution/role.py

```diff
@@ -143,7 +150,7 @@ def build_placeholders(self, obj):
             return type(obj)(r)
         elif isinstance(obj, dict):
             return {key: self.build_placeholders(value) for key, value in obj.items()}
-        elif isinstance(obj, FrameworkTensor):
+        elif isinstance(obj, FrameworkTensor) or isinstance(obj, AbstractTensor):
```

Comment:
I think we should add the Crypten tensor type to `FrameworkTensor`.

Reply:
Moved this part into my PR from here: https://github.com/OpenMined/PySyft/pull/3254